python/metatensor-core/metatensor/core/__init__.py

```diff
@@ -1,13 +1,14 @@
 from .version import __version__  # noqa
 from .block import TensorBlock
-from .labels import Labels
+from .labels import Labels, LabelsEntry
 from .status import MetatensorError
 from .tensor import TensorMap
 
 from .io import load, save
 
 __all__ = [
     "Labels",
+    "LabelsEntry",
```

Comment:
I would not put it in `__all__` (this mainly control what gets imported when the user does `from metatensor import *`)

Reply:
ok
```suggestion
```